es-vue-base/src/lib-mixins/forms.js

```diff
@@ -53,10 +53,19 @@ export default {
         * }
         */
         formErrors() {
-            if (!Object.prototype.hasOwnProperty.call(this.$v, 'form')) {
```

Comment:
Added a bit more error checking so that a page can include the form mixin, but omit a validations section without throwing an error here. In practice this is probably overkill, but the extra defensive-ness here allows form-level validation examples to omit any form-field-level logic which I think is helpful.